### DISCUSSION TITLE
MicroJSON - GeoJSON adapted for microscopy imaging

### DISCUSSION THREAD

**Discussion by bengtl** [developmentseed/geojson-pydantic#163] [Show and tell]
Hello GeoJSON Pydantic community,

We are excited to introduce MicroJSON, a JSON-based format inspired by GeoJSON, tailored for representing microscopy image data such as annotations, regions of interest, and reference points. We have integrated and extended the GeoJSON pydantic classes provided by this repo to meet some of the specific needs of microscopy.  As MicroJSON is fully compatible with GeoJSON, it seamlessly interoperates with existing tools while offering specialized features for microscopy. Specifically, we have for example adapted the OME (Open Microscopy Environment) Data Model for the coordinate system, which may be the most specific difference as compared to geographical applications and standard coordinates.

The official repo can be found at GitHub [MicroJSON repo](https://github.com/polusai/microjson), and the documentation is available at GitHub [Pages for MicroJSON](https://polusai.github.io/microjson/). It is also available at [PyPI](https://pypi.org/project/microjson/).

Would like to thank you all for developing this great repo, which we have been using as a base for our implementation of the pydantic classes of MicroJSON. 
Looking forward to your thoughts and feedback!

Best regards,
Bengt Ljungquist

**Comment by vincentsarago** [comment]
🙏 thanks @bengtl 

This is really cool 🙌
